### DISCUSSION TITLE
Identifying RGB Camera vs Monochrome Cameras

### DISCUSSION THREAD

**Discussion by aurxenon** [bitcraze/discussions#61] [Q&A] [answered]
Hi, the lab I'm working with bought a bunch of monochrome cameras, and then later got an RGB camera as well, however, the camera that I was told is the RGB camera has the same identifier, **608H-1_A** as the known monochrome cameras. Just to confirm, having the marking of **608H-A** means RGB, and **608H-1_A** means monochrome, correct? Semirelated, is there any way to buy a single pair of replacement camera screws for securing the cameras to the AI deck? I accidentally lost one pair and can't find it.

**Comment by knmcguire** [comment, marked as answer]
Hi!

Yes you are right. 608H_A is [color camera ](https://store.bitcraze.io/products/copy-of-ai-deck-color-camera-module) and and 608H-1_A is the [monochrome camera](https://store.bitcraze.io/products/ai-deck-monocrome-camera-module). You can see it on the pictures on the shop (see the links).

We don't sell the screws though, but to quote @tobbeanton: 'It is a M1.4x4 self tapping screw. If you have any toys you could probably find a fitting screw in one of them?'
